libs/blocks/aside/aside.css

```diff
@@ -106,14 +106,9 @@
 
 .aside .foreground.container .text .action-area {
   margin-bottom: 0;
-}
-
-.aside .foreground.container .text .action-area > a {
-  margin-right: var(--spacing-s);
-}
-
-.aside .foreground.container .text .action-area > a:last-child {
-  margin-right: 0;
+  display: flex;
+  gap: var(--spacing-s);
+  flex-wrap: wrap;
```

Comment:
You might also need to add an `align-items: center;` declaration, as the 'Text Link' element is not centred to the CTAs.